scripts/stage-release.sh

```diff
@@ -169,7 +162,17 @@ then
     exit
 fi
 
+# create GitHub release via CLI https://cli.github.com/manual/gh_release_create
+gh release create "${RELEASE_TAG}" ./kroxylicious-*/target/kroxylicious-*-bin.* --title "${RELEASE_TAG}" --notes-file "CHANGELOG.md" --draft
+echo "Draft release created. This must be manually published."
```

Comment:
Hmm I was going to say could we add some colour here so its stands out, but then the goal is to automate this with a GitHub action so its not going to be seen anyway.

Reply:
I'm planning to actually have that published from the release workflow.  So this comment should go away.